content/uiux/concepts/ux-psychology/terms/jacobs-law/jacobs-law.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: "Jacob's Law"
+Description: "Users prefer your site to work like others they already know, because they spend most of their time on other sites."
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+Tags:
+  - 'Design'
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Jacob’s Law**, proposed by Jakob Nielsen, highlights an important principle in UX Design: users spend most of their time on other websites and applications, which shapes their expectations for how things should work. Rather than creating entirely unique designs or interfaces, aligning with familiar patterns and conventions leads to smoother user experiences and less cognitive friction.
+
+This law underlines the importance of consistency and convention in design. Meeting users' expectations based on their existing experiences is how we reduce confusion and increase usability. Unfamiliar or overly original designs often lead to frustration, even if they are innovative or visually appealing.
+
+## Example
+
+A common example of Jacob’s Law in practice is navigation design where most users expect the navigation bar to be at the top of a webpage, with a logo on the left and clickable links for main sections. Moving this navigation to an unexpected place, or using unfamiliar icons without labels, can negatively affect usability.
+
+We help users feel more comfortable and efficient when using our product. It is by designing interfaces that follow patterns users have seen on other platforms.
+
```

Comment:
You can try adding the following sections - 

**1. Why Jacob’s Law Matters**
- Explain its impact on user retention and engagement.
- Highlight how businesses benefit from intuitive design.

**2. Common Applications**
- Examples from popular platforms (e.g., e-commerce sites placing carts in the top-right).
- How mobile apps follow similar design principles across platforms.

**3. Exceptions & When to Innovate**
- When breaking conventions is acceptable (e.g., disruptive innovation, niche audiences).
- Balancing familiarity with creativity.